### ISSUE TITLE
better ui and user friendly features

### ISSUE THREAD

**Issue by callmeVAMP** [JiyaGupta-cs/ShopNex#163]
## Issue Description
[Provide a brief description of the issue]
images of the product are huge. Responsiveness can be improved. Items can be aligned better.
## Steps to Reproduce
[Outline the steps required to reproduce the issue]

## Expected Behavior
[Describe what you expected to happen]

## Actual Behavior
[Describe what actually happened]

## Screenshots
[Include any relevant screenshots]

**Comment by aashkaaaaaa** [comment]
would like to work on this issue, please assign to me.

**Comment by ivedmohan** [comment]
@callmeVAMP can you explain what will you be doing in details ?

**Comment by vaishnavip27** [comment]
Heyyyy @JiyaGupta-cs I would love to work on this issue as a designer and a developer. Please assign me this issue. Looking forward for the approval

**Comment by KashishRathore** [comment]
I would like to make better UI. 
Please assign this to me @ivedmohan 
Can add Bootstrap and Tailwind Css features to make more attractive UI

**Comment by Ketanop321** [comment]
yes the images are way too big , i will adjust it to normal like other E-commerce website   and i will add automatic scroll snap of products that are in offers or latest. I will make overall UI much better and as you said responsiveness is not good its not going work on different media width and also align items more better

**Comment by callmeVAMP** [comment]
The website is good, but there are some bugs. Firstly the images are huge,
even for laptop screen. It doesn't provide better user experience. Sizes
can be made proper. Also scroller can be made more smooth and other
features as well .
